dockerfiles/base/all-tests.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
we are missing a way to call tests from build.sh script. Also do we need the file to be named "all-tests.sh"(or to be in a separate file)